java/java.lsp.server/test/unit/src/org/netbeans/modules/java/lsp/server/ConnectionSpecTest.java

```diff
@@ -29,14 +29,18 @@
 import org.junit.After;
 import org.junit.Test;
 import static org.junit.Assert.*;
+import org.netbeans.junit.NbTestCase;
+import org.netbeans.junit.RandomlyFails;
 
 import org.openide.util.Pair;
 
-public class ConnectionSpecTest {
+@RandomlyFails
+public class ConnectionSpecTest extends NbTestCase {
 
     private static ThreadLocal<List<Boolean>> copySessionServer = new ThreadLocal<>();
 
     public ConnectionSpecTest() {
+        super(ConnectionSpecTest.class.getName());
```

Comment:
Please change the constructor to `ConnectionSpecTest(String testName)` -- each testcase  ought to have an unique name.

Reply:
Done!